sycl/source/detail/graph_impl.cpp

```diff
@@ -528,7 +528,14 @@ exec_graph_impl::enqueue(const std::shared_ptr<sycl::detail::queue_impl> &Queue,
               ->call_nocheck<
                   sycl::detail::PiApiKind::piextEnqueueCommandBuffer>(
                   CommandBuffer, Queue->getHandleRef(), 0, nullptr, OutEvent);
-      if (Res != pi_result::PI_SUCCESS) {
+      if (Res == pi_result::PI_ERROR_INVALID_QUEUE_PROPERTIES) {
+        throw sycl::exception(
+            make_error_code(errc::invalid),
+            "Graphs cannot be submitted to a queue which uses "
+            "immediate command lists. Use "
+            "sycl::ext::intel::property::queue::no_immediate_"
+            "command_list to disable them.");
```

Comment:
```suggestion
            "In the current implementation"
            "Graphs cannot be submitted to a queue using "
            "immediate command lists. E.g. queue property "
            "sycl::ext::intel::property::queue::no_immediate_"
            "command_list is provided for disabling.");
```

Reply:
This suggested change doesn't help clarifying this issue in my opinion. I like the current wording since it is very specific and proposes a clear fix. I think immediate command lists and SYCL Graph contradict each other in general. Adding "In the current implementation" might further confuse users.

Reply:
> This suggested change doesn't help clarifying this issue in my opinion. I like the current wording since it is very specific and proposes a clear fix. I think immediate command lists and SYCL Graph contradict each other in general. Adding "In the current implementation" might further confuse users.

I would agree I don't think this change is necessary.

Reply:
Okay, I'm fine with the error message as is. Feel free to ignore my suggestion.